compliance/repository/src/test/java/org/eclipse/rdf4j/repository/sparql/SPARQLServiceEvaluationTest.java

```diff
@@ -50,8 +50,8 @@
 import org.slf4j.LoggerFactory;
```

Comment:
Are there any `import` rules in the style guide?

Reply:
I believe it has settings to add empty lines between groups. But other things are not set in the current xml file. For example, we have a guideline to not use wildcard imports but this is not enforced by this plugin.